airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/conftest.py

```diff
@@ -126,6 +126,23 @@ def docker_runner_fixture(image_tag, tmp_path) -> ConnectorRunner:
     return ConnectorRunner(image_tag, volume=tmp_path)
 
 
+@pytest.fixture(name="previous_connector_image_name")
+def previous_connector_image_name_fixture(image_tag, inputs) -> str:
+    """Fixture with previous connector image name to use for backward compatibilty tests"""
+    return f"{image_tag.split(':')[0]}:{getattr(inputs, 'previous_connector_version')}"
+
+
+@pytest.fixture(name="previous_connector_docker_runner")
+def previous_connector_docker_runner_fixture(previous_connector_image_name, tmp_path) -> ConnectorRunner:
+    try:
+        return ConnectorRunner(previous_connector_image_name, volume=tmp_path / "previous_connector")
+    except (errors.NotFound, errors.ImageNotFound):
```

Comment:
it seems that we only want to skip if the image not found is `latest`. Otherwise seems like something the developer should know about

Reply:
I updated the logic: return None if the `latest` image could not be found, raise the error if user passed another tag and this tag could not be found -> This will make the backward compatibility test fail as the fixture could not be built.